Update SQLitePCLRaw.lib.e_sqlite3 dependency to 3.50.3

I'm getting a little tired of seeing the following warning all over when I run dotnet build:

> warning NU1603: FwLiteProjectSync depends on SQLitePCLRaw.lib.e_sqlite3 (>= 3.0.3) but SQLitePCLRaw.lib.e_sqlite3 3.0.3 was not found. SQLitePCLRaw.lib.e_sqlite3 3.50.3 was resolved instead.

It should be pretty quick to update our dependency request to 3.50.3. I'm writing this down so I won't forget to do that soon.

--- a/backend/Directory.Packages.props
+++ b/backend/Directory.Packages.props
@@ -119,10 +119,10 @@
     <!-- Only included so it can be excluded. See FwLiteMaui.csproj. -->
     <PackageVersion Include="Mono.Unix" Version="7.1.0-final.1.21458.1" />
     <PackageVersion Include="Soenneker.Utils.AutoBogus" Version="4.0.895" />
-    <PackageVersion Include="SQLitePCLRaw.bundle_e_sqlite3" Version="3.0.3" />
-    <PackageVersion Include="SQLitePCLRaw.core" Version="3.0.3" />
-    <PackageVersion Include="SQLitePCLRaw.lib.e_sqlite3" Version="3.0.3" />
-    <PackageVersion Include="SQLitePCLRaw.provider.e_sqlite3" Version="3.0.3" />
+    <PackageVersion Include="SQLitePCLRaw.bundle_e_sqlite3" Version="3.0.4" />
+    <PackageVersion Include="SQLitePCLRaw.core" Version="3.0.4" />
+    <PackageVersion Include="SQLitePCLRaw.lib.e_sqlite3" Version="3.53.3" />
+    <PackageVersion Include="SQLitePCLRaw.provider.e_sqlite3" Version="3.0.4" />
     <PackageVersion Include="Squidex.Assets.TusClient" Version="8.0.1" />
     <PackageVersion Include="structuremap.patched" Version="4.7.3" />
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="10.1.7" />

